### DISCUSSION TITLE
Purpose of "brew pr-pull" & "brew test-bot" GitHub workflows

### DISCUSSION THREAD

**Discussion by quackerex** [Homebrew/discussions#4913] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

I wanna use a simple brew tap for some personal projects. I created a new tap and it comes with two GitHub workflows. What are the workflows used for and how to I integrate them to my personal tap?

**Comment by SMillerDev** [comment, marked as answer]
If you make a pull request to your tap they will build bottles.

**Reply by quackerex** [reply]
This article also helped me understand it: https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

**Reply by quackerex** [reply]
Also why the name 'pr-pull'? Is it because it is 'pulling' the bottles from the GitHub action?

**Reply by SMillerDev** [reply]
Yes
